src/lex_utils.cc

```diff
@@ -44,11 +44,16 @@ lsPosition CharPos(const std::string& search,
   return result;
 }
 
-bool ShouldRunIncludeCompletion(const std::string& line) {
-  size_t start = 0;
+bool ShouldRunIncludeCompletion(const std::string& line, size_t& start) {
```

Comment:
Pass a pointer, non-const & makes code harder to read IMO since mutation sites are not as apparent.